grader/settings.py

```diff
@@ -177,15 +160,6 @@
 # grader stateful, which at least increases the amount of disk space used.
 ENABLE_PERSONAL_DIRECTORIES = False
```

Comment:
You already removed personal directories in the `access/types/stdasync.py` file, so remove this setting `ENABLE_PERSONAL_DIRECTORIES = False` completely too, please.